src/toil/statsAndLogging.py

```diff
@@ -262,6 +291,8 @@ def add_logging_options(parser: ArgumentParser) -> None:
     group.add_argument("--logFile", dest="logFile", help="File to log in.")
     group.add_argument("--rotatingLogging", dest="logRotating", action="store_true", default=False,
                        help="Turn on rotating logging, which prevents log files from getting too big.")
+    group.add_argument("--no-color", action="store_false", dest="colored_logs", default=True,
+                       help="Enable colored logging")
```

Comment:
The help here seems wrong; `--no-color` would *disable* colored logging.

I would maybe do another `=True/=False` option like all the other new flags we've been adding?

Also, `--no-color` is kebab case while the other Toil option names are usually camelCase. We should probably use camelCase if it is multiple words.